conf/application.conf

```diff
@@ -230,7 +230,13 @@ microservice {
       protocol = "http"
       host = "localhost"
       port = 9570
-      maximumFileSize = 23068672  # 22 MB
+      maximumFileSize = 20000000  # 20 MB
+    }
+
+    common-transit-convention-traders {
+        protocol = "http"
+        host = "localhost"
+        port = 9487
```

Comment:
This is a bit recursive!

I wonder if there is a better way to get this -- I wonder what other services that need their own URL do. Might be worth a search.

Reply:
Agreed! I'll look into it

Reply:
As discussed, the only solution I found is less clean than this one :)